superset/assets/src/SqlLab/App.jsx

```diff
@@ -56,7 +56,10 @@ const sqlLabPersistStateConfig = {
         if (path === 'sqlLab') {
           subset[path] = {
             ...state[path],
+            databases: {},
```

Comment:
should we keep databases around? there aren't that many of them usually

Reply:
👍